p2panda-rs/src/test_utils/README.md

```diff
@@ -0,0 +1,114 @@
+<h1 align="center">p2panda-rs test utils</h1>
```

Comment:
This high-level overview into testing is a little buried inside the `test_utils` module. Could parts of it be moved into the `p2panda-rs` `README.md`? Another solution could be to move it all into module-level Rust documentation, so at least it will show in the generated documentation then :+1:

Reply:
nice!